common/src/main/java/io/netty/util/internal/InternalThreadLocalMap.java

```diff
@@ -92,15 +79,12 @@ public static void remove() {
         if (thread instanceof FastThreadLocalThread) {
             ((FastThreadLocalThread) thread).setThreadLocalMap(null);
         } else {
-            ThreadLocal<InternalThreadLocalMap> slowThreadLocalMap = UnpaddedInternalThreadLocalMap.slowThreadLocalMap;
-            if (slowThreadLocalMap != null) {
-                slowThreadLocalMap.remove();
-            }
+            slowThreadLocalMap.remove();
         }
     }
 
     public static void destroy() {
-        slowThreadLocalMap = null;
+        slowThreadLocalMap.remove();
```

Comment:
@normanmaurer It's to prevent `slowThreadLocalMap` from being initialized again after its destruction. Otherwise a user can 'resurrect' the destroyed `InternalThreadLocalMap`.